### DISCUSSION TITLE
Project name

### DISCUSSION THREAD

**Discussion by jtyoung84** [AllenNeuralDynamics/aind-data-transfer#63] [Q&A] [answered]
@carshadi @dyf @alejoe91 

 - Are we fine with the current name of the project? The others are suffixed as aind instead of nd.
 - I'm also not too familiar with PyPI. Should the "transfer" package be renamed to "nd_data_transfer" (or whatever the project name is)?
 -  Should we break apart the dependencies for the ophys and ephy requirements?

**Comment by dyf** [comment, marked as answer]
My votes:

- yes rename to `aind-data-transfer`
- I believe the convention is dashes in the package name `aind-data-transfer` and underscores in the code `import aind_data_transfer`
- yes split apart so we can do a `pip install aind-data-transfer[ephys]` or `pip install aind-data-transfer[imaging]`

**Reply by dyf** [reply]
I think we have consensus here, opening tickets and marking this as accepted.

**Comment by alejoe91** [comment]
Same as @dyf for me!
